wire-compiler/src/main/java/com/squareup/wire/WireCompiler.kt

```diff
@@ -98,6 +98,11 @@ import okio.openZip
  *
  * The `--android-annotations` flag will add the `Nullable` annotation to optional fields.
  *
+ * The `opaque_types` takes a list of qualified named Protobuf types separated by a ','. All types
```

Comment:
nit maybe add `\`--opaque_types\`` to match the rest?

Reply:
fixed!